Schema/app/002-initial.blue.sql

```diff
@@ -4,20 +4,25 @@ INSERT INTO odin.migration VALUES('app', '002-initial.blue.sql');
 
 -- APP
 CREATE TABLE odin.app (
-    app_id TEXT NOT NULL CHECK (odin.url_safe(app_id)),
+    app_id TEXT NOT NULL,
     app_name TEXT NOT NULL,
+    token TEXT NOT NULL,
     access_policy TEXT NOT NULL,
     data_sharing_policy TEXT NOT NULL,
     redirect_url TEXT NOT NULL,
     changed TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
     created TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
-    PRIMARY KEY (app_id)
+    PRIMARY KEY (app_id),
```

Comment:
Should put these with the fields that they are talking about. This line and the ones below for the foreign key should go here under the `app_id` column.

Reply:
Disagree, I think this style is much easier for reading and reasoning.

Reply:
You're wrong :-) But let's discuss offline. Short argument is that nowhere else in the code would you want to put closely related items far apart from each other.